Automattic/wp-calypso
# Remove extra illustrions from JPO

This PR removes extra illustrations that were renamed in #22957.

**Testing instructions:**

- Checkout this branch
- Verify that these images have been removed from /images/illustrations.
